e2e-test/src/test/java/de/sovity/edc/extension/e2e/PostgresFlywayExtensionTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2023 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *      sovity GmbH - init
+ */
+
+package de.sovity.edc.extension.e2e;
+
+import de.sovity.edc.extension.e2e.connector.Connector;
+import de.sovity.edc.extension.e2e.connector.JsonLdConnectorUtil;
+import de.sovity.edc.extension.e2e.connector.factory.TestConnectorFactory;
+import de.sovity.edc.extension.e2e.db.TestDatabase;
+import de.sovity.edc.extension.e2e.db.TestDatabaseFactory;
+import jakarta.json.JsonObject;
+import org.eclipse.edc.junit.extensions.EdcExtension;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.Map;
+import java.util.UUID;
+
+import static de.sovity.edc.extension.e2e.connector.config.EdcApiType.PROTOCOL;
+import static jakarta.json.Json.createObjectBuilder;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.eclipse.edc.connector.transfer.spi.types.TransferProcessStates.COMPLETED;
+import static org.eclipse.edc.jsonld.spi.JsonLdKeywords.CONTEXT;
+import static org.eclipse.edc.jsonld.spi.JsonLdKeywords.TYPE;
+import static org.eclipse.edc.jsonld.spi.PropertyAndTypeNames.ODRL_POLICY_ATTRIBUTE;
+
+@Disabled("Just to be executed with postgres-flyway property set")
```

Comment:
What does this mean?

Reply:
It means that the flyway-extension needs to be compiled into the connector-jar and there needs to be a postgres-container running to execute this test.

I would like to separate this into another issue: Adapting gh actions to spin up a postgres container und run those tests